include/nvexec/stream/schedule_from.cuh

```diff
@@ -41,6 +41,7 @@ namespace nvexec::STDEXEC_STREAM_DETAIL_NS {
         template < __completion_tag Tag, class... As>
         friend void tag_invoke(Tag, __t&& self, As&&... as) noexcept {
           storage_t* storage = static_cast<storage_t*>(self.operation_state_.temp_storage_);
+          ::new (storage) storage_t();
```

Comment:
Please, add a TODO to remove host access later. Also, could you have an optimization for the base case `sizeof...(as) == 0` to just propagate tag without touching temporary storage?